docs/source/tutorial-contributing.rst

```diff
@@ -117,6 +117,7 @@ List of tutorial participants
 - Samia Razvi (UT Dallas)
 - Nicoke Cuneo (Haskins Laboratories)
 - Kristine Hocker (MIT)
+- Estelle Hervé (AMU)
 
```

Comment:
- Priyanka Shreedar (UC Davis)

Reply:
@pshreedar You'll actually want to make your own separate PR to try it out, this is an existing one for another tutorial participant. Let me know or post in the Slack #tutorial channel if you have any trouble!